packages/create-react-native-library/templates/nitro-module/ios/{%- project.name %}.swift

```diff
@@ -0,0 +1,10 @@
+class <%- project.name -%>: Hybrid<%- project.name -%>Spec {
+    public var hybridContext = margelo.nitro.HybridContext()
+    public var memorySize: Int {
+        getSizeOf(self)
+    }
```

Comment:
I should really work on getting this lifted into the spec instead of having it at user class level. Maybe Nitro 19